src/components/DryRun/DryRun.tsx

```diff
@@ -0,0 +1,169 @@
+import Grid from "@dashboard/components/Grid";
+import { useStyles } from "@dashboard/custom-apps/components/WebhookEvents/styles";
+import {
+  useTriggerWebhookDryRunMutation,
+  WebhookEventTypeAsyncEnum,
+} from "@dashboard/graphql";
+import {
+  capitalize,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Typography,
+} from "@material-ui/core";
+import {
+  Alert,
+  Button,
+  DialogHeader,
+  List,
+  ListBody,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+} from "@saleor/macaw-ui";
+import React, { useState } from "react";
+import { useIntl } from "react-intl";
+
+import DryRunItemsList from "../DryRunItemsList/DryRunItemsList";
+import { messages } from "./messages";
+import { getObjects } from "./utils";
+
+interface DryRunProps {
+  query: string;
+  showDialog: boolean;
+  setShowDialog: React.Dispatch<React.SetStateAction<boolean>>;
+  asyncEvents: WebhookEventTypeAsyncEnum[];
+  setResult: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const DryRun = (props: DryRunProps) => {
```

Comment:
suggestion: I also like this approach to typing functional components but in dashboard common is to use React.FC